docs/fsharp/tools/fsharp-interactive/index.md

```diff
@@ -117,6 +117,9 @@ printfn "%A" (f (dsharp.tensor 1.2))
 
 You can specify as many package references as you like in a script.
 
+> [!WARNING]
```

Comment:
```suggestion
> [!NOTE]
```

I would keep this as a `NOTE`. The warning is typically better for when doing something could have a destructive outcome.

Reply:
got it, thanks